mne_bids_pipeline/_config.py

```diff
@@ -706,6 +725,16 @@
 parameters.
 """
 
+notch_trans_bandwidth: float = 1.0
```

Comment:
This is now inconsistent with the other trans bandwidth settings. Is that intentional?

Reply:
Yes there is no `'auto'` for `notch_filter`

https://mne.tools/dev/generated/mne.io.Raw.html#mne.io.Raw.notch_filter

Reply:
Oh, right!